Android.mk

```diff
@@ -17,6 +17,9 @@ LOCAL_PATH:= $(call my-dir)
 TA_PATH:= $(LOCAL_PATH)
 include $(CLEAR_VARS)
 
+# Extra flags for SDP usage
+USE_TEE_SDP ?= fasle
```

Comment:
This should be CFG_SECURE_DATA_PATH to keep it consistent with other TA's like Playready (also it follows OP-TEE's nomenclature.

I think you can also make generating the secureDecryptor dependent on this flag